proto/proto/penumbra/core/transaction/v1alpha1/transaction.proto

```diff
@@ -156,6 +157,13 @@ message ActionView {
     dex.v1alpha1.PositionWithdraw position_withdraw = 32;
     dex.v1alpha1.PositionRewardClaim position_reward_claim = 34;
 
+    stake.v1alpha1.Delegate delegate = 41;
+    stake.v1alpha1.Undelegate undelegate = 42;
+    // TODO: we have no way to recover the opening of the undelegate_claim's
+    // balance commitment, and can only infer the value from looking at the rest
+    // of the transaction. is that fine?
```

Comment:
For instance, in a transaction that splits the undelegate claim into two sub-actions (not sure why they'd do that, but nothing stops them), we can't see how much was in each, we can only infer the net amount. 

We also can't recover the full opening of the balance commitment in any case, because we don't retain the blinding factor, and commitments are perfectly hiding.

This seems like it's probably fine, or that to the extent it's not, it's not worth the cost of "fixing".